regular/talks.html

```diff
@@ -59,6 +59,7 @@ <h3><em>Can computers compile complex code correctly?</em> by Matt Windsor</h3>
 <h3><em>Communicate! Pair Programming, sketchnotes and compassion</em> by Miki Mokrysz</h3>
 <b>November 21st (Autumn Week 8)</b>
 <p>
+<b><em>This talk will probably not take place this term. We apologise for the inconvience and are trying to reschedule it</em></b>
```

Comment:
Seems good except for 3 minor points:

1. "probably" it definitely won't take place so that should probably be removed
2. "inconvience" should be "inconvenience"
3. There should probably be a full stop at the end of the line

Here's my suggested version:

```suggestion
<b><em>This talk will not take place this term. We apologise for the inconvenience and are trying to reschedule it</em>.</b>
```